README.md

```diff
@@ -111,6 +111,39 @@ If you’d like to add a request proxy for the API client (to avoid CORS issues)
 </script>
 ```
 
+#### Events [beta]
+
+We have recently added two events to the standalone CDN build only.
+
+##### scalar:reload-references
+
+Reload the references, this will re-mount the app in case you have switched pages or the dom
+elements have been removed.
+
+```ts
+document.dispatchEvent(new Event('scalar:reload-references'))
+```
+
+##### scalar:reload-references
```

Comment:
I think the heading should say `scalar:update-references-config`. ✌️

Reply:
nice catch!